mern/client/src/Contexts/sessionProvider.js

```diff
@@ -6,10 +6,18 @@ export default function SessionProvider({ children }) {
   // nav state to show or hide session id/save/delete buttons
   const [navState, setNavState] = React.useState(false);
   const [transcriptID, setTranscriptID] = React.useState();
+  const [firstQuestions, setfirstQuestions] = React.useState();
```

Comment:
Also, add a comment here saying what the state is/it's purpose. Easier for understanding.

Reply:
Good idea!